docs/source/R-api.rst

```diff
@@ -12,165 +12,6 @@ You can use the R API to `install MLflow`_, start the `user interface <Run MLflo
     :local:
     :depth: 1
 
-Crate a function to share with another process
```

Comment:
Note that changes to .rst and .Rd files are autogenerated, with the .Rd files ultimately generating R-api.rst. Will make a follow-up PR to add the .Rd files to gitignore so they don't bloat diff size